tests/rt.conf

```diff
@@ -276,13 +276,16 @@ COMPILE | datm_cdeps_faster | intel | -DAPP=NG-GODAS -DFASTER=ON  | - wcoss2
 RUN | datm_cdeps_control_cfsr_faster                    | - wcoss2                             | baseline |
 
 ### CDEPS Data Atmosphere tests with LND ###
-COMPILE | datm_cdeps_land | intel | -DAPP=LND | - wcoss2    | fv3 |
-RUN | datm_cdeps_lnd_gswp3                              | - wcoss2                             | baseline |
-RUN | datm_cdeps_lnd_gswp3_rst                          | - wcoss2 noaacloud                   |          | datm_cdeps_lnd_gswp3
+COMPILE | datm_cdeps_land | intel | -DAPP=LND | - wcoss2 | fv3 |
+RUN | datm_cdeps_lnd_gswp3                              | - wcoss2           | baseline |
+RUN | datm_cdeps_lnd_era5                               | - wcoss2           | baseline |
+RUN | datm_cdeps_lnd_era5_rst                           | - wcoss2 noaacloud |          | datm_cdeps_lnd_era5
 
-### ATM-LND tests, -D32BIT=ON has issue and NoahMP reuires r8 libraries ###
-COMPILE | atml | intel | -DAPP=ATML -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v17_p8,FV3_GFS_v15_thompson_mynn_lam3km | - noaacloud    | fv3 |
-RUN | control_p8_atmlnd_sbs                             | - noaacloud                          | baseline |
+### ATM-LND tests ###
+COMPILE | atml | intel | -DAPP=ATML -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v16_flake,FV3_GFS_v17_p8,FV3_GFS_v17_p8_rrtmgp,FV3_GFS_v15_thompson_mynn_lam3km,FV3_WoFS_v0,FV3_GFS_v17_p8_mynn -D32BIT=ON | | fv3 |
```

Comment:
Curious about the different SDFs you've listed. Are you using anything other than the GFS_V17_P8?  There are 3 flavors of v17_p8 tests listed, so I'm not sure which one is actually used.

Reply:
@DeniseWorthen I checked the `tests/tests/control_p8_atmlnd` and it seems that i am using `FV3_GFS_v17_p8` and I think this is consistent with `control_p8`.